Source/WebCore/accessibility/AXCoreObject.h

```diff
@@ -1630,7 +1630,6 @@ T* findChild(T& object, F&& matches)
 {
     for (auto child : object.children()) {
         if (matches(child)) {
-            RELEASE_ASSERT(is<T>(child.get()));
             return downcast<T>(child.get());
```

Comment:
You need to use `checkedDowncast<>()` here so you get the same RELEASE_ASSERT() on all platforms.